lualib/resty/events/worker.lua

```diff
@@ -104,6 +125,98 @@ function _M.new(opts)
     return setmetatable(self, _MT)
 end
 
+local function read_thread(self, broker)
+    while not terminate(self) do
+        local data, err = broker:recv_frame()
```

Comment:
`broker` may not be a good name, I prefer `conn`, it means a connection to broker, not broker itself.

Reply:
`broker_connection` then.